### ISSUE TITLE
Poll Blocks on P2: Results only show first 5 responses

### ISSUE THREAD

**Issue by cuemarie** [Automattic/wp-calypso#62439]
### Quick summary

I put together a number of Poll Blocks on a P2 post for a team game, and 8 people voted in the polls, but only 5 results were tracked across each poll block.

Not sure if this is a unique issue for P2s, or because the post had multiple Poll Blocks set up in a row (9 total).

### Steps to reproduce

These are the exact steps I took:

1. Create a Post on P2
2. Set up 9 Poll Blocks with 9 answers each (each poll block was nested inside a cover block)
3. Published Post
4. Had 8 teammates vote using the poll blocks

### What you expected to happen

Blocks were set to show results once a vote was submitted. That worked for everyone, and the expectation is that the results would show 8 votes tallied.

### What actually happened

After submitting a vote, results did display for each participant, but only the first 5 votes were included in the results. This is what everyone participating saw.

Example of Poll block results after 8 people voted: 

<img width="663" alt="Markup 2022-03-31 at 13 03 07" src="https://user-images.githubusercontent.com/27249804/161139997-256e0dc8-cf55-48c8-9bfc-83525844c4be.png">

### Context

Private Team P2 where this occurred: p9aQKR-1Ze-p2

There's a few factors at play here that might have impacted this:

- Poll Block on P2
- Multiple Poll Blocks in one post
- Poll Blocks nested inside cover blocks

### Simple, Atomic or both?

Simple

### Theme-specific issue?

**P2**

### Reproducibility

Consistent

### Severity

Some (< 50%)

### Available workarounds?

No but the platform is still usable

### Workaround details

Because this was on a P2, connecting to Crowdsignal to view results seems like it won't work, since ownership of P2 sites works a little differently.

**Comment by inaikem** [comment]
I tried to replicate with multiple Poll blocks on both a "normal" private Simple site and private P2 (pc4f5j-ip-p2).

@cuemarie, are you still able to replicate this on your end? If so, let's get this under the eyes of the Crowdsignal and P2 folks to get a better understanding of where the issue lies!

**Comment by cuemarie** [comment]
Thanks @inaikem ! Returning to the Poll I built this issue for, p9aQKR-1Ze-p2 , I found that I need to submit my answers again to see the results - which makes me wonder if the Polls timed out or something after a certain amount of response (we were all live voting at one time - perhaps there's something going on there?)

When I resubmit a vote this time, the poll records my vote as the 6th vote: 

![Markup on 2022-04-21 at 07:19:54](https://user-images.githubusercontent.com/27249804/164478675-0e03a896-0c63-4da9-82e4-bd6371522f4b.png)

So the problem isn't _exactly_ there anymore, but evidence of it remains. For some reason, the poll stopped recording at 5 votes at the time we were using it - across everyone's different machines. We could all see the results, which means the poll registered that we voted, but didn't save our votes at the time.

Does that help? :D
